docs/content/installation.md

```diff
@@ -20,4 +20,15 @@ git clone https://github.com/grafana/grizzly.git
 cd grizzly
 make dev
 sudo mv grr /usr/local/bin/grr
-```
\ No newline at end of file
+```
+
+Installing Grizzly on macOS via Homebrew
```

Comment:
Could you make this a heading? H2 I figure. And add an "Installing Grizzly on Linux" heading further up?

Reply:
Added headers, added language for fenced blocks, and reordered document.